src/structures/ThreadChannel.js

```diff
@@ -423,15 +425,10 @@ class ThreadChannel extends Channel {
    */
   get sendable() {
     return (
-      !this.archived &&
-      (this.type !== 'private_thread' || this.joined || this.manageable) &&
-      this.permissionsFor(this.client.user)?.any(
-        [
-          Permissions.FLAGS.SEND_MESSAGES,
-          this.type === 'GUILD_PRIVATE_THREAD'
-            ? Permissions.FLAGS.USE_PRIVATE_THREADS
-            : Permissions.FLAGS.USE_PUBLIC_THREADS,
-        ],
+      !(this.archived && this.locked && !this.manageable) &&
+      (this.type !== 'GUILD_PRIVATE_THREAD' || this.joined || this.manageable) &&
+      this.permissionsFor(this.client.user)?.has(
```

Comment:
you need to account for `this.permissionsFor(...)` returning null, if its null the getter will return undefined

Reply:
The getter would return false because its in a giant boolean logic chain and null or undefined == false right?

Reply:
actually i think sugden is correct here; it's the last one in the chain so if the first 2 are falsy, the value of the last one will be returned

Reply:
oh right there's no Boolean(), yeah